js/popup.js

```diff
@@ -154,10 +159,14 @@ window.onload = function() {
         }
 
         var special = '';
-        if(document.getElementById('adv_meanings').checked !== true) {
+        if (document.getElementById('adv_meanings').checked !== true) {
             special = '&d=1';
         }
 
+        var safe = '&kp=1';
```

Comment:
@Sayanc93 Is there a good reason for having another variable (`safe` in this case) when there already is `special` in place?